test/metabase/api/card_test.clj

```diff
@@ -2960,3 +2962,33 @@
                           (-> (jdbc/query (sql-jdbc.conn/connection-details->spec driver/*driver* details)
                                           ["SELECT EXISTS (SELECT 1 FROM information_schema.tables WHERE table_schema = 'public')"])
                               first :exists))))))))))
+
+(deftest csv-upload-snowplow-test
+  (mt/test-driver :h2
+    (mt/with-empty-db
+      (let [db-id (u/the-id (mt/db))]
+        (mt/with-temporary-setting-values [uploads-enabled      true
+                                           uploads-database-id  db-id
+                                           uploads-schema-name  "PUBLIC"
+                                           uploads-table-prefix nil]
+          (snowplow-test/with-fake-snowplow-collector
+            (upload-example-csv! nil)
+            (is (=? {:data {"model_id"        pos?
+                            "size_mb"         3.910064697265625E-5
```

Comment:
```suggestion
                            "size_mb"         pos?
```

where does this number come from? and do we really need to care about its exact value?